dbt/adapters/spark_cde/cdeapisession.py

```diff
@@ -284,7 +284,7 @@ def getJobOutput(self, job):
 
         no_of_retries = 0
 
-        while len(res_lines) == 0:
+        while len(res_lines) <= 3:  # ensure that enough lines are present to start parsing
```

Comment:
Suggest to make it like MIN_LINES_FOR_PARSING insead of 3 directly

Reply:
updated.